worker/worker.go

```diff
@@ -217,6 +222,24 @@ func processCampaign(c *models.Campaign) {
 	}
 }
 
+// Creates a simple constant delay between sending each email message.  Later
```

Comment:
Golang code style suggests to use the function name to start the code documentation. So something like:
"SendingDelay creates a simple...". Also, go ahead and trim out the "Later..." since we don't want this in the official docs that are rendered. You can add this to the commit message.